cmd/olympus/actions/goget.go

```diff
@@ -0,0 +1,37 @@
+package actions
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gobuffalo/buffalo"
+	"github.com/gomods/athens/pkg/cdn"
+)
+
+// GoGet is middleware that checks for the 'go-get=1' query string. If it exists,
+// uses getter to determine the redirect location
+func GoGet(getter cdn.Getter) buffalo.MiddlewareFunc {
```

Comment:
Does olympus need goget support? go tooling should communicate with proxy, let's keep communication flow simple.

Reply:
Yea, because go tooling needs to be able to talk to Olympus too